Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "dapr"
-version = "0.7.0"
+version = "0.7.1-alpha.0"
```

Comment:
Only a point release? Not a minor version bump?

Reply:
It has been my preference to use patch releases (`z` of `0.y.z`) for non-breaking features pre 1.0.  Apparently, there is no set rule for SemVer on this...  https://github.com/semver/semver/issues/333.  If would prefer to use `0.8.0-alpha.0` instead, I'm OK with that.